Quadriga/src/main/webapp/WEB-INF/views/workspace/workspacedetails.jsp

```diff
@@ -382,10 +388,38 @@
 </div>
 <!-- /.modal -->
 
+<!-- Changing Text Accessibility modal -->
+<div class="modal fade" tabindex="-1" role="dialog" id="changeaccess-ws">
+	<div class="modal-dialog">
+		<div class="modal-content">
+			<div class="modal-header">
+				<button type="button" class="close" data-dismiss="modal"
+					aria-label="Close">
+					<span aria-hidden="true">&times;</span>
+				</button>
+				<h4 class="modal-title">Update Text Accessibility</h4>
+			</div>
+			<div class="modal-body">
+				<p>Are you sure you want to change accessibility of text file?</p>
```

Comment:
This should say: "Are you sure you want to make this text file [public|private]?"  depending on the action, it should say 'public' or 'private'.